open-ce/validate_config.py

```diff
@@ -26,52 +26,58 @@ def make_parser():
                                description = 'Perform validation on a conda_build_config.yaml file.')
     return parser
 
-def validate_config(arg_strings=None):
-    '''
-    Entry function.
-    '''
+def _main(arg_strings=None):
     args = make_parser().parse_args(arg_strings)
     variants = utils.make_variants(args.python_versions, args.build_types, args.mpi_types)
+    validate_env_config(args.conda_build_config, args.env_config_file, variants, args.repository_folder)
+
+def validate_env_config(conda_build_config, env_config_files, variants, repository_folder):
+    '''
+    Validates a lits of Open-CE env files against a conda build config
+    for a given set of variants.
+    '''
     for variant in variants:
-        print('Validating {} for {}'.format(args.conda_build_config, variant))
-        for env_file in args.env_config_file:
-            print('Validating {} for {} : {}'.format(args.conda_build_config, env_file, variant))
+        print('Validating {} for {}'.format(conda_build_config, variant))
+        for env_file in env_config_files:
+            print('Validating {} for {} : {}'.format(conda_build_config, env_file, variant))
```

Comment:
We can remove this print statement.

Reply:
Or may be above one?

Reply:
Yeah, I think we should remove most of the print statements now that this will be run during each build.

Reply:
I removed all but the starting and finished statements.